poc/architectures/transformer.py

```
@@ -0,0 +1,123 @@
+        )
+
+
+def check_ollama_available(model: str = "llama3.1:8b") -> bool:
+    """Check if Ollama is running and model is available."""
+    try:
+        import ollama
+
+        models = ollama.list()
+        available = [m["name"] for m in models.get("models", [])]
+        return any(model in m for m in available)
+    except Exception:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)